src/dao/ERC20ManagerDAO.js

```diff
@@ -20,7 +20,7 @@ const EVENT_TOKEN_ADD = 'LogAddToken'
 const EVENT_TOKEN_MODIFY = 'LogTokenChange'
 const EVENT_TOKEN_REMOVE = 'LogRemoveToken'
 
-const NON_OPTIONAL_TOKENS = ['ETH', 'TIME', 'BTC', 'BCC']
+const NON_OPTIONAL_TOKENS = ['ETH', 'TIME', 'BTC', 'BCC', 'BTG', 'LTC']
```

Comment:
stay only erc20 here

Reply:
Anton,
I asked you to rename ERC20ManagerDAO a long, long time ago.
I made a description for example here:
https://chrono-dev.slack.com/archives/C74R4QBR7/p1505914508000067

So,
Won't fix

But the ERC20ManagerDAO is really need to be refactored and renamed.
Yu know it.